format.md

```diff
@@ -49,51 +49,72 @@ The terminology for array types in this section is based on the
 GeoArrow proposes a packed columnar data format for the fundamental geometry
 types, using packed coordinate and offset arrays to define geometry objects.
 
-The inner level is always a FixedSizeList array storing the interleaved
-coordinate values (`[x, y, x, y, ...]`, physically stored as a single array
-of length `n_dim * n_coords`). For any geometry type except Point, this inner
-level is nested in one or multiple
+The inner level is always an array of coordinates. For any geometry type except
+Point, this inner level is nested in one or multiple
 [variable sized list](https://arrow.apache.org/docs/format/Columnar.html#variable-size-list-layout)
 arrays. In practice, this means we have additional arrays storing the offsets
 that denote where a new geometry, a new geometry part, or a new polygon ring
 starts.
 
-**Point**: `FixedSizeList<double>[n_dim]`
+This specification supports coordinates encoded as a Struct array storing the
+coordinate values as separate arrays (i.e., `x: [x, x, ...], y: [y, y, y, ...]`)
+and a FixedSizeList of interleaved values (i.e., `[x, y, x, y, ...]`). As
+implementations evolve, this specification may grow to support other coordinate
+representations or shrink to support only one if supporting multiple
+representations becomes a barrier to adoption.
 
-For an array of Point geometries, only a single array is used of interleaved
-coordinates. `n_dim` can be 2, 3, or 4 depending on the dimensionality of the
-geometries, and the field name of the list should be "xy", "xyz", "xym" or
-"xyzm", reflecting the coordinate interpretation.
+**Coordinate (Struct)**: `Struct<x: double, y: double, [z: double, [m: double>]]`
```

Comment:
```suggestion
**Coordinate (separated)**: `Struct<x: double, y: double, [z: double, [m: double>]]`
```

Do we want to use the "separated" term when talking about the non-interleaved layout for the coordinates? 
(a bit of a nitpick, but it's good to have clear and consistent terminology we can use throughout)

While "struct" is the name of the type in Arrow, it has the disadvantage that it can be slightly ambiguous in general (some might consider the interleaved layout actually an "array of structs", in context of the SoA vs AoS terms). And we still clearly list this layout to use a Struct type on the remainder of the line.

We already use "separated" in the paragraph above for the explanation of the struct-based layout, and I saw that @kylebarron is also using this term in the geoarrow-rs implementation (https://github.com/kylebarron/geoarrow-rs/blob/032f6a7099e9a8a3f2c1d0fe84a178e886e3e5b9/src/array/coord/combined/array.rs#L8-L24)